### DISCUSSION TITLE
How to connect with capability json files?

### DISCUSSION THREAD

**Discussion by Blatman** [tauri-apps/tauri#9168] [Q&A] [answered]
Really struggling to convert from V1 to V2 but basically I did a migration on my project and a migrated.json file was created in the capabilities folder. This file appears to hold all the old allowlist details so somehow this needs to be linked into the tauri.config.json file (which used to hold the allowlist stuff) - I was thinking maybe that it needs to go under the security section but it is not clear in the docs. Any clues anyone?

**Comment by FabianLars** [comment, marked as answer]
iirc tauri is supposed to read all capability files by default. Being able to select them in tauri.conf was added her: https://github.com/tauri-apps/tauri/pull/8797 (not updated in the docs yet). So you could add the identifier from the respective capability file ([example](https://github.com/tauri-apps/tauri/blob/dev/examples/api/src-tauri/capabilities/main.json#L3)) to the app > security > capabilities array if you prefer that.

**Comment by Blatman** [comment]
@FabianLars  thanks for that - it does look like Tauri is using the migrated.json file so just need to find out why the fs api is failing (** doesn't appear to work so need to look further)
